### PR TITLE
[frontend] Cannot add tag to a dashboard/investigation (#5861)

### PR DESCRIPTION
**Pull request by SarahBocognano**
### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/5861

### CODE REVIEW

**Comment by Kedae** [comment]
Can we have a small e2e test for that ? :)

**Comment by CelineSebe** [comment]
it works

File: opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { expect, test } from "../fixtures/baseFixtures";
+import { DashboardPage } from "../model/dashboard.pageModel";
```
Comment (frapuks) [inline opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts]: All tests e2e are verified by Linter since Landry PR.

**Review by frapuks — COMMENTED** [review]
Just linters issues, but feature work perfectly

**Review by marieflorescontact — COMMENTED** [review]
locally tested ok 👌

**Review by Kedae — APPROVED** [review]

**Pull request merged**